src/webview/MessageList.js

```diff
@@ -290,10 +298,13 @@ class MessageList extends Component<Props> {
     return (
       <WebView
         useWebKit
+        startInLoadingState
+        renderLoading={this.renderLoading}
```

Comment:
For the commit message, one perhaps-helpful phrase in the linked docs is "show the loading view on the first load." I think that helps clarify what it's meant to do.

In particular if in

    Use startInLoadingState and renderLoading props to present a blank,
    transparent View, during the initial "loading state."

we say

    [...] during "the first load."

I think that's already somewhat more reassuring. I agree it's still not the world's clearest documentation.

Tiny nit, but I also feel like I want a paragraph break between that sentence and what comes after. The first is saying what we do, and the rest is offering a caveat and expanding on that with some details.